media/css/m24/components/navigation-refresh.scss

```diff
@@ -221,16 +221,18 @@ $margin-top: 58px; // top margin offset for mobile navigation menu
         h2::after {
             @include bidi(((margin-left, $spacing-sm, margin-right, 0),));
             content: '';
-            background: url('/media/img/icons/caret-sprite.svg') no-repeat bottom left / auto 12px;
+            background: url('/media/protocol/img/icons/arrow-down.svg') no-repeat center center / auto 16px;
             display: inline-block;
             height: 12px;
             vertical-align: baseline;
             width: 12px;
+            transform-origin: center;
+            transition: transform 0.25s ease;
```

Comment:
Ideally these would only apply based on prefers-reduced-motion media query…

Reply:
Hmm I did remove the transition on the chevron rotation, but forgot about the slower arrow movement on the orange category link hover…

Do we know a good best practice on frequency/length etc. that might be preferable to avoid, and/or if some movement is OK… or I should better follow up with removing any movement on the arrow hover to be on the safe side?